products/batch_exports/backend/temporal/destinations/redshift_batch_export.py

```diff
@@ -1212,6 +1306,12 @@ async def copy_into_redshift_activity_from_stage(inputs: RedshiftCopyActivityInp
             [field.with_nullable(True) for field in record_batch_schema if field.name != "_inserted_at"]
         )
 
+        # Redshift recommends files should be between 1MB-1GB, and that we
+        # should aim for files to be equally distributed across cluster slices.
+        # We don't know the number of slices in our user's cluster, so I've chosen
+        # some nice round somewhere in the range (100MB).
+        # TODO: Maybe derive this from user's input?
```

Comment:
This should be good enough so that larger batch exports can benefit from parallelization in Redshift COPY, even if the resulting files are not evenly distributed across cluster slices.